Add support for Symfony 7.0

jerowork opened the pull request:
# Description
Update `composer.json` to support Symfony `^7.0`.

jerowork committed:
Add support for Symfony ^7.0

ruudk commented:
@Yozhef could you approve the workflow, merge it, and tag it 🙏 Thanks

Yozhef commented on composer.json:
```diff
@@ -16,11 +16,11 @@
     "require": {
         "php":                   "^7.4|^8.0",
-        "symfony/css-selector":  "^4.4|^5.0|^6.0"
+        "symfony/css-selector":  "^4.4|^5.0|^6.0|^7.0"
```
Please add to CI process to run Symfony version 7.0

jerowork replied:
Sharp, added 7.0 to be tested for PHP8.2

jerowork committed:
Add SF 7.0 to GH actions

Excluding all PHP versions lower than 8.2, as Symfony 7.0 minimum requirement
is PHP 8.2.

jerowork commented:
Thanks @Yozhef 🎉

Yozhef commented:
@jerowork thank you very much for the contribution and development - sorry for taking so long to watch
